packages/platforms/android-player/src/ts/classes/android-machine.ts

```diff
@@ -54,7 +54,7 @@ export class AndroidMachine implements Machine {
         latitude: location.coords.latitude,
         longitude: location.coords.longitude,
       };
-    } catch (e) {
+    } catch {
```

Comment:
wouldn't it be better to cast the e argument to Error and throw that instead?

Reply:
According to the machine interface we should return undefined if no location can be determined.

Reply:
wouldn't we at least want to log out these errors? otherwise it would difficult to understand why it goes wrong when it goes wrong.

Reply:
Added logging of this error (for all platforms)